### ISSUE TITLE
About dropout

### ISSUE THREAD

**Issue by pitera05** [philipxjm/Neural-Stock-Market-Prediction#8]
Hi 
@ line 103, may I know if it's typo? since the keep_prob will be 0 when dropout is 1, it's strange.

          if dropout > 0.0:
            output = tf.nn.dropout(output, keep_prob=1 - dropout)
